### PR TITLE
[ig/security] Avoid quoting the W3C's mission.

### PR DESCRIPTION
**Pull request by jyasskin**
Edit: This PR used to have the charter copy the mission. After discussion, it now avoids trying to quote the mission, since the mission might change.

### CODE REVIEW

**Review by tantek — CHANGES_REQUESTED** [review]
The W3C Mission is still in-progress and I disagree with an attempt to precisely quote it in a charter (which will get frozen once approved). 

Instead, mention something higher level, and replace that whole paragraph with something shorter like:

> W3C's Mission(linked) includes privacy and security for the web.

Then simplify the next sentence (in the next paragraph) like this:

s/The last two requirements, Privacy and Security, are integral to/Both of these are integral to/

Then collapse those two paragraphs into one to simplify and shorten the charter.

File: 2024/ig-security.html [hunk 1]
```diff
@@ -148,7 +148,7 @@ <h1 id="title">PROPOSED Security Interest Group Charter</h1>
 
       <div id="background" class="background">
         <h2>Motivation and Background</h2>
-        <p>W3C’s <a href="https://www.w3.org/mission/">mission</a> is to <q>develop open standards for an interoperable web to meet requirements for accessibility, internationalization, privacy, and security – so that humanity can experience all that the web has to offer</q>.</p>
+        <p>W3C’s <a href="https://www.w3.org/mission/">mission</a> is to “develop open standards for an interoperable web to meet requirements for accessibility, internationalization, privacy, and security – for technologies which connect and empower humanity”.</p>
```
Comment (tantek) [inline 2024/ig-security.html]:
```suggestion
        <p>W3C’s <a href="https://www.w3.org/mission/">mission</a> includes privacy and security for the web.
```

File: 2024/ig-security.html [hunk 2]
```diff
@@ -148,7 +148,7 @@ <h1 id="title">PROPOSED Security Interest Group Charter</h1>
...
         <p>The last two requirements, Privacy and Security, are integral to <a href="https://www.un.org/en/about-us/universal-declaration-of-human-rights" title="Article 3: Everyone has the right to life, liberty and security of person.">human rights</a> and <a href="https://www.ohchr.org/en/instruments-mechanisms/instruments/international-covenant-civil-and-political-rights" title="Article 9: Everyone has the right to liberty and security of person">civil liberties</a> and have always been of the Consortium's concern.</p>
```
Comment (tantek) [inline 2024/ig-security.html]:
```suggestion
        Both of these are integral to <a href="https://www.un.org/en/about-us/universal-declaration-of-human-rights" title="Article 3: Everyone has the right to life, liberty and security of person.">human rights</a> and <a href="https://www.ohchr.org/en/instruments-mechanisms/instruments/international-covenant-civil-and-political-rights" title="Article 9: Everyone has the right to liberty and security of person">civil liberties</a> and have always been of the Consortium's concern.</p>
```

**Comment by jyasskin** [comment]
@tantek's change is fine with me. The mission was quoted at all because of @mnot's comment in the [previous review](https://www.w3.org/2002/09/wbs/33280/sing/results), so he should get a chance to weigh in.

**Comment by mnot** [comment]
I was only concerned with it being misquoted / mischaracterised.

**Comment by tantek** [comment]
@mnot wrote:
> I was only concerned with it being misquoted / mischaracterised.

Do my proposed changes (in https://github.com/w3c/charter-drafts/pull/609#pullrequestreview-2379185442 and subsequent suggested changes) address those concerns and/or can you live with my proposed changes?

**Review by simoneonofri — APPROVED** [review]

**Comment by mnot** [comment]
They're not saying what the mission **is**, so I think you avoid the original issue.

**Comment by tantek** [comment]
@jyasskin wrote:
> @tantek's change is fine with me.

Thanks. If you can accept my changes into this PR I'll re-review and approve.

**Comment by simoneonofri** [comment]
@jyasskin, @tantek, and @mnot, thank you for the useful discussion. The thinking here, too, might be important for making charters more resilient to various changes.

@jyasskin I was thinking about the usage of `<q>` vs. the normal one with `"`
I am waiting for the acceptance and review to Merge and close.

**Comment by jyasskin** [comment]
I've merged @tantek's suggested changes, and retitled the PR to match.

**Review by tantek — APPROVED** [review]

**Pull request merged**